xds/src/main/java/io/grpc/xds/XdsLoadBalancerProvider.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds;
+
+import io.grpc.Internal;
+import io.grpc.LoadBalancer;
+import io.grpc.LoadBalancer.Helper;
+import io.grpc.LoadBalancerProvider;
+
+/**
+ * The provider for the "xds" balancing policy.  This class should not be directly referenced in
+ * code.  The policy should be accessed through {@link io.grpc.LoadBalancerRegistry#getProvider}
+ * with the name "xds" (currently "xds_experimental").
+ */
+@Internal
+public final class XdsLoadBalancerProvider extends LoadBalancerProvider {
+
+  @Override
+  public boolean isAvailable() {
+    return true;
+  }
+
+  @Override
+  public int getPriority() {
+    return 5;
+  }
+
+  @Override
+  public String getPolicyName() {
+    return "xds_experimental";
```

Comment:
Can we define this as a constant and use everywhere? The other place is XdsLoadBalancerTest.java

Reply:
I agree to use a constant for a string used multiple places in the main code. But here this is the only one place in the main code. Using the raw string in test is more readable and helps find typo in the string constant.